### ISSUE TITLE
Write new backend API based on new [TIOF] Project Lockdown [I] Dataset Layer structure EN v1.0

### ISSUE THREAD

**Issue by daphnecys** [TheIOFoundation/ProjectLockdown#2]
1. Create the new classes first.

**Comment by JFQueralt** [comment]
Environments
https://theiofoundation.stoplight.io/docs/projectlockdown/API/Reference/Project-Lockdown.v1.json/paths/~1Environments~1%7BDSLUID%7D~1%7BModule%7D~1%7BEnvironmentUID%7D/get


Territories

**Comment by JFQueralt** [comment]
@ColisaNo195 @SalocinDotTEN 
Have you distributed who takes what?

**Comment by jeff-knurek** [comment]
each "object" created in mongodb comes with a unique ObjectId: https://docs.mongodb.com/manual/reference/method/ObjectId/
this unique value is referenced by `_id` field of the object, and should be documented to be used as the `UID` for each model.

for the endpoint queries, this `_id` value will be the default choice for use in query strings 

---
for any case where a sequential 1,2,3.. id should be used, that should be clearly documented

**Comment by SalocinDotTEN** [comment]
> @ColisaNo195 @SalocinDotTEN
> Have you distributed who takes what?

Yes, sort of. I deal with the endpoints starting with "Data..." and she deals with the other 3.

**Comment by ColisaNo195** [comment]
Agree on the distribution.

In the sample JSON generated by Stoplight, $ref is expanded into an entire object. Can anyone suggest how should references look like in sample JSON? So that for any object documented as $ref, we can replace the object in sample JSON with proper reference.

**Comment by SalocinDotTEN** [comment]
Hi, the ambiguous errors encountered in Stoplight.

![image.png](https://images.zenhubusercontent.com/5ffbf91ff41307c610de2e23/a485b24b-251f-45c0-a8cf-b34d20e98002)
Project-Lockdown.v1.json issues in Stoplight.

![image.png](https://images.zenhubusercontent.com/5ffbf91ff41307c610de2e23/c92ae698-b50c-413e-ba81-dded91ce529c)
API/api.yaml errors in Stoplight.

![image.png](https://images.zenhubusercontent.com/5ffbf91ff41307c610de2e23/d12bfd6a-92d8-4eed-9bd0-8a222f4fae18)
DEI/api.yaml errors in Stoplight.
